example/lib/main.dart

```diff
@@ -170,6 +173,40 @@ class _MyAppState extends State<MyApp> {
       print("Failed to open document: '${e.message}'.");
     }
   }
+
+  void showFormDocumentExample() async {
+    try {
+      final ByteData bytes =
+          await DefaultAssetBundle.of(context).load(_formPath);
+      final Uint8List list = bytes.buffer.asUint8List();
+
+      final tempDir = await getTemporaryDirectory();
+      final tempDocumentPath = '${tempDir.path}/$_formPath';
+
+      final file = await File(tempDocumentPath).create(recursive: true);
+      file.writeAsBytesSync(list);
+
+      Pspdfkit.present(tempDocumentPath);
+
+      Pspdfkit.setFormFieldValue("Lastname", "Name_Last");
+      Pspdfkit.setFormFieldValue("0123456789", "Telephone_Home");
+      Pspdfkit.setFormFieldValue("City", "City");
+      Pspdfkit.setFormFieldValue("selected", "Sex.0");
+      Pspdfkit.setFormFieldValue("deselected", "Sex.1");
+      Pspdfkit.setFormFieldValue("selected", "HIGH SCHOOL DIPLOMA");
+
+      String lastName;
+      // Platform messages may fail, so we use a try/catch PlatformException.
+      try {
+        lastName = (await Pspdfkit.getFormFieldValue("Name_Last")) as String;
+        print(lastName);
```

Comment:
Instead of silently printing the last name in the example, I would add a button which, when tapped, will show an alert with the value of the form field.

This should look similar to what we do in our React Native example. See https://github.com/PSPDFKit/react-native/blob/master/samples/Catalog/Catalog.ios.js#L860-L870

![recording](https://user-images.githubusercontent.com/7443038/62778195-c167e280-ba7d-11e9-9f44-613630f370d8.gif)